svmbir/svmbir.py

```diff
@@ -203,18 +203,19 @@ def auto_sigma_prior(sino, delta_channel = 1.0, sharpness = 0.0 ):
     return sigma_prior
 
 
-def auto_num_rows(num_channels, delta_channel, delta_pixel):
-    """Compute the automatic value of ``num_rows``.
-    """
-    num_rows = int(np.ceil(num_channels * delta_channel / delta_pixel))
-    return num_rows
+def auto_img_size(geometry, num_channels, delta_channel, delta_pixel):
+    "Compute the default image size"
 
+    if geometry == 'parallel':
+        num_rows = int(np.ceil(num_channels * delta_channel / delta_pixel))
+    elif geometry == 'fan':
+        num_rows = num_channels
```

Comment:
In the parallel beam case, the image size depends on the channel and pixel sizes, but in the fan beam case it seems to be fixed.

What if we do `num_rows = int(np.ceil(num_channels * delta_channel / (delta_pixel*magnification)))` ?

Reply:
Oh yeah, I was thinking the delta_channel and delta_pixel as set to the defaults. I'll make the change.